README.md

```diff
@@ -173,7 +175,11 @@ Once the database is created, the dashboard will update to show the documents in
 
 Click `Create Document` to create/save the document.
 
-* Repeat the above steps and create documents for the remaining grievances as shown in [SampleData/MyWardGrievances.json](SampleData/MyWardGrievances.json).
+* Repeat the above steps and create documents for the remaining sample grievances: [SampleData/MyWardGrievance2.json](SampleData/MyWardGrievance2.json), [SampleData/MyWardGrievance3.json](SampleData/MyWardGrievance3.json), [SampleData/MyWardGrievance4.json](SampleData/MyWardGrievance4.json), [SampleData/MyWardGrievance5.json](SampleData/MyWardGrievance5.json), [SampleData/MyWardGrievance6.json](SampleData/MyWardGrievance6.json).
+
+The `myward` database should now list the six documents as shown below under `Table` view.
+
+<img src="doc/source/images/CloudantDatabasePopulated.png" alt="Cloudant database populated with sample data" width="800" border="10" />
```

Comment:
just use the simpler markdown syntax:

  `![](doc/source/images/CloudantDatabasePopulated.png)`

Also, no image called CloudantDatabasePopulated.png exists yet, so this renders as a broken link

Reply:
By mistake, I checked in the required image directly to master branch: https://github.com/IBM/Ionic-MFP-App/tree/master/doc/source/images . Hence we won't have the broken link problem on master branch.

The simpler markdown syntax doesn't allow us to shrink image (in my case I am shrinking by specifying width=800). I wanted to show a reasonably sized image in Readme.md yet wanted to allow users to open the image in a new tab to see finer details in case they want to.

Reply:
I see it in the master branch. gotcha.